share/icons/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ if (OS_IS_MAC)
 
     install(FILES MsczIcon/MS4_MsczIcon.icns RENAME MsczIcon.icns DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME})
     install(FILES MscxIcon/MS4_MscxIcon.icns RENAME MscxIcon.icns DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME})
+    install(FILES cloud_icon.svg DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME})
```

Comment:
You should move these `install` commands out of the platform `if` statements and only write it once since it's the same on all platforms.

Reply:
yeah makes sense. Updated the PR.